static/src/uu/index.scss

```diff
@@ -1,7 +1,22 @@
-@import "variables";
 @import "font";
 @import "badge";
 
-img.logo {
-    height: 48px;
-}
+// Dark theme specific settings
+[data-bs-theme="dark"] {
+    /* Breadcrumb */
+    .breadcrumb {
+        --bs-breadcrumb-bg: var(--bs-black);
+        --bs-breadcrumb-divider-color: var(--bs-gray-alt2);
+        --bs-breadcrumb-item-active-color: var(--bs-gray-alt);
+    }
+    
+    /* Links */

```

Comment:
This fixes the colors of the data tables, but for example the front page buttons contrast is bad. We can porbably make the CSS selector more specific.
![Screenshot 2023-08-18 at 15-31-02 Yoda - Dev ‐ Home](https://github.com/UtrechtUniversity/yoda-portal/assets/29246264/1748005a-b104-40cb-844e-d726d4cd91dd)